### PR TITLE
auth

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
 - Added login and signup page

## Screenshot:
image
image

### CODE REVIEW

File: auth/login.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            </li>
+        </ul>
+    </nav>
+
+    <div class="w-100 flex-row content-center mt-4">
+        <form class="login-form py-2 px-4 w-100 card-shadow rounded-sm m-2" action="./login.html">
+            <h3 class="text-center mt-2 mb-3">Login</h3>
+            <div class="flex-column mb-3">
+                <label for="email">Email address</label>
+                <input class="mt-1 py-1 px-2 text-base border rounded-sm" id="email" type="email"
```
Comment (sanket4120) [inline auth/login.html:68]: You can create a utility classes for different types of input instead of repeating different spacing classes

**Review by sanket4120 — APPROVED** [review]
looks good to me

File: auth/login.html (hunk 1 above)
Comment (rohanmathur91) [reply auth/login.html:68]: I have discussed this issue with our mentor, so when we work in react we only loop over the single component so there's no need for copy-paste, that's the reason to not create a utility for that.

**Review by HarshithaKonduru214 — APPROVED** [review]
Looks good.

**Pull request merged**